ci-templates/gitlab/README.rst

```diff
@@ -0,0 +1,35 @@
+..
+   SPDX-FileCopyrightText: Copyright DB Netz AG and the capellambse contributors
+   SPDX-License-Identifier: Apache-2.0
+
+Gitlab CI templates
+===================
+
+Currently, we provide the following Gitlab CI/CD templates:
+
+- `Model badge`_: Produces and pushes a complexity model badge.
+
+Model badge
+-----------
+Please add the following section to your `.gitlab-ci.yml`:
+
+.. code:: yaml
+
+    include:
+    - remote: https://raw.githubusercontent.com/DSD-DBS/py-capellambse/${PY_CAPELLAMBSE_REVISION}/ci-templates/gitlab/model-badge.yml
+
+    variables:
+    ENTRYPOINT: test/test.aird # Entry point to the .aird file of the model (relative from root level of the repository)
+
+    # The following section is only needed if you want to change the output filename
+    generate-model-badge:
+    variables:
+        OUTPUT_FILE: model-complexity-badge.svg # Change the filename here
+
+In addition, you have to add the following environment variables on Gitlab project or group level.
+Make sure to enable the "Expand variable reference" flag.
+
+- `PY_CAPELLAMBSE_REVISION`: Revision of this Github repository
```

Comment:
This also needs to mention the other relevant (mandatory) CI variables, namely GIT_USERNAME and GIT_PASSWORD.